kiwi_lint/model_form.py

```diff
@@ -0,0 +1,23 @@
+from pylint import interfaces
+from pylint import checkers
+from pylint_django.utils import node_is_subclass
+
+
+class ModelFormChecker(checkers.BaseChecker):
+    """
+    Check to ensure that all forms inherit from ModelForm.
+    """
+    __implements__ = (interfaces.IAstroidChecker,)
+
+    name = 'model-form-checker'
+
+    msgs = {'R4811': ('All forms must inherit from ModelForm',
+                      'inherit-from-model-form',
+                      'All forms must inherit from ModelForm'
+                      'See: https://github.com/kiwitcms/Kiwi/issues/1384')}
+
+    def visit_classdef(self, node):
+        if node.name.endswith('Form') and not node_is_subclass(node,
```

Comment:
all forms in Django seem to inherit from django.forms.BaseForm. Instead of searching by name which may be unreliable in some cases try checking if the current node subclasses `BaseForm` but doesn't subclass `ModelForm`. That should catch all possible forms in the app which aren't ModelForm-s.